aten/src/ATen/native/native_functions.yaml

```diff
@@ -2546,6 +2546,11 @@
   dispatch:
     CPU: quantize_linear_cpu
 
+- func: quantize_linear_per_channel(Tensor self, Tensor scales, Tensor zero_points, Tensor axis, ScalarType dtype) -> Tensor
```

Comment:
why  `axis` is a 1-d tensor and not `int[]` like other sizes?

Reply:
was trying to be consistent.. I will change to `int[]`